### ISSUE TITLE
[Perl] Implement new Concept Exercise: basics

### ISSUE THREAD

**Issue by mienaikage** [exercism/v3#2297]
This issue describes how to implement the `basics` concept exercise for the Perl track.

## Getting started

**Please please please read the docs before starting.** Posting PRs without reading these docs will be a lot more frustrating for you during the review cycle, and exhaust Exercism's maintainers' time. So, before diving into the implementation, please read up on the following documents:

- [The features of v3](https://github.com/exercism/v3/blob/master/docs/features-of-v3.md).
- [Rationale for v3](https://github.com/exercism/v3/blob/master/docs/rationale-for-v3.md).
- [What are concept exercise and how they are structured?](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md)

Please also watch the following video:

- [The Anatomy of a Concept Exercise](https://www.youtube.com/watch?v=gkbBqd7hPrA).

## Goal

This exercise should teach the bare minimum to help students understand Perl programs.

## Learning objectives

- Know of the strict/warnings pragmas.
- Know how to declare a `my` scalar variable (`my $var`).
- Know how to assign a variable.
- Know how to declare a named subroutine.
- Know how to return from a subroutine.
- Know how to call a subroutine.
- Know how to comment.

## Out of scope

- Package.
- Export.
- Version pragma.
- Array/hash variables.
- Other variable declarations (`our`/`local`).
- Subroutine arguments/signatures (`@_`).

## Concepts

- `basics`: know of the strict/warnings pragmas; know how to declare a `my` scalar variable; know how to assign a variable; know how to declare a named subroutine; know how to return from a subroutine; know how to call a subroutine; know how to comment.

## Prerequisites

N/A

## Resources to refer to

https://perldoc.perl.org/perlintro

## Help

If you have any questions while implementing the exercise, please post the questions as comments in this issue.

**Comment by mienaikage** [comment]
I've excluded subroutine arguments from basics as I think having an arrays/lists exercise should probably come before introducing e.g. `my $self = shift` / `my ($self) = @_`
